package.json

```diff
@@ -2,10 +2,21 @@
   "name": "koa-body",
   "version": "5.0.0",
   "description": "A Koa body parser middleware. Supports multipart, urlencoded and JSON request bodies.",
-  "main": "index.js",
-  "types": "./index.d.ts",
```

Comment:
we don't need to specify `types`?

Reply:
since we are a typescript project now, we export typings along side with our files and therefore typescript can resolve them without mentioning them in the package.json
<img width="379" alt="Bildschirmfoto 2022-10-24 um 16 02 50" src="https://user-images.githubusercontent.com/7534085/197544743-eb51e086-5c0a-45c7-b4e0-5797e2018631.png">

Reply:
the types keyword would be there so non-typescript projects' IDEs could know where to find type info
I'll test this out locally

Reply:
@TheDadi yeah let's re-add the types property please
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
done